packages/mds-policy-service/service/provider.ts

```diff
@@ -99,5 +100,16 @@ export const PolicyServiceProvider: ServiceProvider<PolicyService, PolicyService
       }
 
       return publishedPolicy
+    }),
+  writePolicyIntentToPolicy: (context, intent_draft) =>
+    serviceErrorWrapper('writePolicyIntentToPolicy', async () => {
+      const policy = translateIntentToPolicy(intent_draft)
+      const { policy_id } = policy
+      await PolicyRepository.writePolicy(policy)
```

Comment:
Would this work assuming the repository returns the policy?
```suggestion
      const { policy_id } = await PolicyRepository.writePolicy(translateIntentToPolicy(intent_draft)
)
```